app/views/stafftools/assignment_repos/_assignment_repo.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="flex-table classroom-resource">
+  <div class="flex-table-item">
+    <span class="assignment-icon assignment-icon-individual left">
+      <%= octicon('repo') %>
+    </span>
+
+    <div class="student-information">
+      <%= link_to stafftools_assignment_repo_path(assignment_repo.id) do %>
+        <strong class="css-truncate-target login">Assignment Repo: <%= assignment_repo.id %></strong>
+      <% end %>
+    </div>
+
```

Comment:
When you're working on the other views, try and provide status info to show that things are in a good (expected state) or bad state. For example showing that a user's token is invalid on the user stafftools page.